imports/plugins/core/shipping/server/methods/methods.js

```diff
@@ -62,6 +62,40 @@ export const methods = {
         });
       }
     }
+  },
+
+  /**
+   * @method shipping/updateParcelSize
+   * @summary update defaultParcelSize
+   * @param {Object} parcel - size to be updated
+   * @since 2.0.0
+  */
+  "shipping/updateParcelSize"(parcel) {
+    check(parcel, {
+      weight: Match.Optional(Number),
+      height: Match.Optional(Number),
+      length: Match.Optional(Number),
+      width: Match.Optional(Number)
+    });
+
+    if (!Reaction.hasPermission(shippingRoles)) {
+      throw new Meteor.Error(403, "Access Denied");
```

Comment:
Please follow the documentation for how to throw errors https://docs.reactioncommerce.com/reaction-docs/master/error-handling-guide